gherkin-tests/src/world.rs

```diff
@@ -0,0 +1,114 @@
+use std::{collections::BTreeMap, convert::Infallible, str::FromStr, sync::Arc};
+
+use async_trait::async_trait;
+use cucumber::{Parameter, WorldInit};
+use many_client::client::ledger::{LedgerClient, Symbol, TokenAmount};
+use many_client::ManyClient;
+use many_identity::{Address, CoseKeyIdentity};
+
+use crate::{cose::new_identity, opts::SpecConfig};
+
+#[derive(Parameter, Debug, Hash, Ord, PartialOrd, Eq, PartialEq)]
+#[param(regex = r"[\w\d]+", name = "identity")]
+pub struct IdentityName(String);
+
+impl FromStr for IdentityName {
+    type Err = String;
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Ok(IdentityName(s.to_string()))
+    }
+}
+
+#[derive(Debug, WorldInit)]
+pub struct World {
+    spec_config: Option<Arc<SpecConfig>>,
+    identities: BTreeMap<IdentityName, CoseKeyIdentity>,
+    symbols: BTreeMap<String, Symbol>,
+    client: Option<ManyClient>,
+    ledger_client: Option<LedgerClient>,
+}
+
+impl World {
+    pub fn client(&self) -> &ManyClient {
+        self.client.as_ref().unwrap()
+    }
+
+    pub fn ledger_client(&self) -> &LedgerClient {
+        self.ledger_client.as_ref().unwrap()
+    }
+
+    pub async fn init_config(&mut self, spec_config: Arc<SpecConfig>) {
+        self.client = Some(
+            ManyClient::new(
+                spec_config.server_url.clone(),
+                spec_config.ledger_identity.identity,
```

Comment:
You can make this `Address::anonymous()`.

Or use the ledger identity if and only if the `ledger.pem` file is present.

See https://github.com/liftedinit/many-rs/pull/186#discussion_r954958609